etc/confd/templates/env.tmpl

```diff
@@ -12,10 +12,12 @@ DBNAME={{getv "/postgres/default/dbname"}}
 DBUSER={{getv "/postgres/default/username"}}
 DBPASSWORD={{getv "/postgres/default/password"}}
 
-TEST_SERVER_URL="http://127.0.0.1:8000"
+TEST_SERVER_URL="http://{{getv "/url/addr"}}"
 TEST_URL=""
```

Comment:
Why keep this var

Reply:
.env vars should be cleaned up (see selenium acceptance tests https://github.com/pia-lab/pialab-back/blob/master/tests/_support/WebGuy.php#L47)